test/functional/specs/C2588.js

```diff
@@ -23,14 +12,18 @@ test.meta({
   TEST_RUN: "Regression"
 });
 
-test("Test C2588: Throw error when configure is executed multiple times.", async t => {
-  await configureAlloyInstance("alloy", config);
+const configureAlloyInstance = ClientFunction(config => {
+  return window
+    .alloy("configure", config)
+    .then(() => {}, error => error.message);
+});
 
-  await configureAlloyInstance("alloy", config);
+test("Test C2588: Throw error when configure is executed multiple times.", async t => {
+  await configureAlloyInstance(orgMainConfigMain);
+  const errorMessage = await configureAlloyInstance(orgMainConfigMain);
 
-  const { error } = await t.getBrowserConsoleMessages();
```

Comment:
This was testing that an error was logged to the console rather than testing that the command's promise was actually rejected.

Reply:
Nice!